pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -47,15 +58,14 @@ Panels::GSWindowSettingsPanel::GSWindowSettingsPanel( wxWindow* parent )
 	m_check_HideMouse	= new pxCheckBox( this, _("Always hide mouse cursor") );
 	m_check_CloseGS		= new pxCheckBox( this, _("Hide window when paused") );
 	m_check_Fullscreen	= new pxCheckBox( this, _("Default to fullscreen mode on open") );
-	m_check_VsyncEnable	= new pxCheckBox( this, _("Wait for Vsync on refresh") );
 	m_check_DclickFullscreen = new pxCheckBox( this, _("Double-click toggles fullscreen mode") );
 	m_check_AspectRatioSwitch = new pxCheckBox(this, _("Switch to 4:3 aspect ratio when an FMV plays"));
 	//m_check_ExclusiveFS = new pxCheckBox( this, _("Use exclusive fullscreen mode (if available)") );
 
 	m_text_Zoom->SetToolTip( pxEt( L"Zoom = 100: Fit the entire image to the window without any cropping.\nAbove/Below 100: Zoom In/Out\n0: Automatic-Zoom-In untill the black-bars are gone (Aspect ratio is kept, some of the image goes out of screen).\n  NOTE: Some games draw their own black-bars, which will not be removed with '0'.\n\nKeyboard: CTRL + NUMPAD-PLUS: Zoom-In, CTRL + NUMPAD-MINUS: Zoom-Out, CTRL + NUMPAD-*: Toggle 100/0"
 	) );
 
-	m_check_VsyncEnable->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
+	m_combo_vsync->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
```

Comment:
> read your link ;)

I did. But I got lost once everybody in the fucking world started to have a **very** convinced opposite opinion 😭😭
At least in the small little theory I have in mind though, it sounds as best as [v-sync] can: render as fast as possible, only present most recent frame. 
Of course (I think?) this would only make a difference whenever rendering (and not only presentation) is in the slim sub-60-yet-still-full-speed limbo. 

> Tbh, the only good solution is gsync/freesync.

Oh, absolutely, that's 100% perfection. 
But in that case we really wouldn't have bother with anything in the first place 😛 

> Dunno how PAL behaves. Likely badly. I'm curious if we can emulate faster to render at 60 fps. 

That's pretty defeatist on your side
Aside of nobody actually even ever complaining about this.. We could have an option that (when in fullscreen - when we'll have it) switches screen to 50Hz (or even 59.94 ?) - which at least in my monitor are natively supported refresh rates. 

> Yes it would be better to automatically detect compositor but I'm not sure how to do it. 

.-.
Didn't we had like 50 posts just to discuss that?
Putting aside the nvidia case (which I already suggested a dumb workaround, to be used just until hopefully they can answer me) it seems to just be about DwmIsCompositionEnabled && window!=monitor

... Or were you referring specifically to the actual windows functions to call? :S

 > Besides option is nice.

Only if your users are smart enough! (and option makes sense separated in the first place)

> Having a way to enable/disable this flag will greatly help the testing

Ok, makes sense, for as much at the moment I'm thinking to either some style we still aren't using or driver bug.